extensions/adobe/experience/customerJourneyManagement/message-delivery-feedback.schema.json

```diff
@@ -104,6 +104,27 @@
           },
           "meta:titleId": "message-delivery-feedback##https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/messageFailure##title##88741"
         },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/source": {
+          "title": "Message Source",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/sourceType": {
+              "title": "Source Type",
+              "type": "string",
+              "description": "Classifies the source type of the message."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/sourceId": {
+              "title": "Source Id",
+              "type": "string",
+              "description": "Source Id of the entity where message originate."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/messageDeliveryfeedback/sourceActionId": {
+              "title": "Source ActionId",
+              "type": "string",
+              "description": "Action node id of the source from where message originate"
```

Comment:
We should remove "node" from the description, as we would be ingesting a actionId not nodeId(In AJO journey nodeId is different from actionId)